src/amo/components/LandingPage.scss

```diff
@@ -1,5 +1,58 @@
 @import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
 
 .LandingPage {
   padding: $padding-page;
 }
+
+.LandingPage-header {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin: $padding-page $padding-page ($padding-page * 2);
+}
+
+.LandingPage-header-top {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin-bottom: $padding-page * 2;
+}
+
+.LandingPage-header-text {
+  @include margin-start(20px);
+
+  display: inline-block;
+  max-width: 350px;
```

Comment:
How does this look on tablets/desktop/larger screens? Where does this number come from?